app/javascript/apps/ArtsyPullRequests/ArtsyPullRequests.spec.tsx

```diff
@@ -1,9 +1,7 @@
 import React from "react"
 import { act } from "react-dom/test-utils"
-import { mount } from "enzyme"
+import { render, screen, waitFor } from "@testing-library/react"
```

Comment:
You can kind of think of `render` like enzyme's mount. Render is a bit magical though because it's not a one off operation. It's continually rendering to the `screen` property as changes happen. You don't have to do things like call `update` like enzyme would do. Most of that stuff just happens for you... just like if you were running react in a webpage.